fleetshard/main.go

```diff
@@ -62,6 +66,123 @@ func main() {
 		}
 	}()
 
+	glog.Info("Creating certMonitor")
+
+	certmonitorConfig := &certmonitor.Config{
+
+		Monitors: []certmonitor.MonitorConfig{
+			{
+				Namespace: certmonitor.SelectorConfig{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "rhacs.redhat.com/tenant",
+								Operator: metav1.LabelSelectorOpExists,
+							},
+						},
+					},
+				},
+				Secret: certmonitor.SelectorConfig{ // pragma: allowlist secret
+					Name: "scanner-tls",
+				},
+			},
+			{
+				Namespace: certmonitor.SelectorConfig{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "rhacs.redhat.com/tenant",
+								Operator: metav1.LabelSelectorOpExists,
+							},
+						},
+					},
+				},
+				Secret: certmonitor.SelectorConfig{ // pragma: allowlist secret
+					Name: "central-tls",
+				},
+			},
+
+			{
+				Namespace: certmonitor.SelectorConfig{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "rhacs.redhat.com/tenant",
+								Operator: metav1.LabelSelectorOpExists,
+							},
+						},
+					},
+				},
+				Secret: certmonitor.SelectorConfig{ // pragma: allowlist secret
+					Name: "scanner-db-tls",
+				},
+			},
+
+			{
+				Namespace: certmonitor.SelectorConfig{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "rhacs.redhat.com/tenant",
+								Operator: metav1.LabelSelectorOpExists,
+							},
+						},
+					},
+				},
+				Secret: certmonitor.SelectorConfig{ // pragma: allowlist secret
+					Name: "scanner-v4-db-tls",
+				},
+			},
+
+			{
+				Namespace: certmonitor.SelectorConfig{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "rhacs.redhat.com/tenant",
+								Operator: metav1.LabelSelectorOpExists,
+							},
+						},
+					},
+				},
+				Secret: certmonitor.SelectorConfig{ // pragma: allowlist secret
+					Name: "scanner-v4-indexer-tls",
+				},
+			},
+			{
+				Namespace: certmonitor.SelectorConfig{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "rhacs.redhat.com/tenant",
+								Operator: metav1.LabelSelectorOpExists,
+							},
+						},
+					},
+				},
+				Secret: certmonitor.SelectorConfig{ // pragma: allowlist secret
+					Name: "scanner-v4-matcher-tls",
+				},
+			},
+		},
+	}
+
+	if errs := certmonitor.ValidateConfig(*certmonitorConfig); len(errs) != 0 {
+		glog.Fatalf("certmonitor validation error: %w", errs)
```

Comment:
I think in this case it's 
```suggestion
		glog.Fatalf("certmonitor validation error: %v", errs)
```